### PR TITLE
docs(theme): Get Started in Style

### PR DESCRIPTION
**Pull request by paulftw**
Fixes GQLSAAS-741

Screen Shot 2020-10-13 at 12 28 11 PM

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6713)

 
 

Docs Preview: [Dgraph Preview](https://dgraph-3511eefc04-101326.surge.sh)

### CODE REVIEW

File: wiki/content/_index.md [hunk 1]
```diff
@@ -9,162 +9,134 @@ aliases = ["/contribute"]
...
+    <div class="icon"><i class="fa fa-book" aria-hidden="true"></i></div>
+    <a href="https://dgraph.io/tour/" target="_blank">
+      <h2>Tour of Dgraph</h2>
+      <p>
+        Take an interactive tour of Dgraph to learn the concepts
+      </p>
+    </a>
+  </div>
+  <div class="item">
+    <div class="icon"></div>
```
Comment (bucanero) [inline wiki/content/_index.md:66]: Hi @paulftw ,
could you add an icon to the Tutorials card? thanks!

**Review by bucanero — CHANGES_REQUESTED** [review]
The new page looks great.

I left a comment to add a missing icon.

Also, the title styling for `Clients`, `Deploy`, `Enterprise features`, `FAQ`, and `Badger` cards look different than the rest.  Other titles are black and bold, but these are regular and blue. I assume it's just a CSS style fix.

File: wiki/content/_index.md (hunk 1 above)
Comment (paulftw) [reply wiki/content/_index.md]: done

**Comment by paulftw** [comment]
@bucanero addressed your comments.

In this PR I only focused on getting the designer's style codified. 
Microcopy, icons, section order, etc is something that DevRel or Docs teams can modify any time they like.

**Comment by bucanero** [comment]
> @bucanero addressed your comments.
> 
> In this PR I only focused on getting the designer's style codified.
> Microcopy, icons, section order, etc is something that DevRel or Docs teams can modify any time they like.

thanks @paulftw , yes you're right, we can take it from here. Thanks again for your help!

**Pull request merged**